### ISSUE TITLE
[TW-1402] Outdated taskwarrior cygwin package

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1429]
_Stefan Betz on 2014-08-27T23:18:23Z says:_ 

 The Taskwarrior cygwin package is outdated since some time, atm task 2.2.0 is in the repos where on linux 2.3.0 is released.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-08-27T23:18:23Z
Modified: 2015-03-14T18:00:38Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-08-31T20:53:42Z says:_ 

 It's all about manpower.  But this should be addressed with a 2.4.0 release.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-03-14T18:00:38Z says:_ 

 Taskwarrior 2.4.1 has 32-bit and 64-bit Cygwin packages available.
